spec/features/web_helper.rb

```diff
@@ -1,8 +1,15 @@
 def successful_sign_up
-  visit "/users/new"
+  visit "/"
   fill_in 'First name', with: "test_first_name"
   fill_in 'Last name', with: "test_last_name"
   fill_in 'Email', with: "test_email@keepingitrails.com"
   fill_in 'Password', with: "pa55w0rd"
   click_button "Sign Up"
 end
+
+def unsuccessful_sign_up_with_wrong_password
```

Comment:
let's have web helpers for all the sign in scenarios
- successful_sign_in
- sign_in_with_incorrect_email
- sign_in_with_incorrect_password